asserts/ifacedecls.go

```diff
@@ -150,6 +150,14 @@ func (matcher mapAttrMatcher) feature(flabel string) bool {
 
 func (matcher mapAttrMatcher) match(apath string, v interface{}, ctx AttrMatchContext) error {
 	switch x := v.(type) {
+	case Attrer:
+		// we get Atter from root-level Check (apath is "")
+		for k, matcher1 := range matcher {
```

Comment:
The variable names `matcher` and `matcher1` are confusing. Could you rename `matcher1` to perhaps just `m` here? Alternatively call the method receiver `m` and use matcher here but I guess it's not any better.

Reply:
it's the preexisting style of the package that was followed, I don't have super strong feelings either way but is followed material if we care enough